### ISSUE TITLE
Buffer deprecation warning during package install

### ISSUE THREAD

**Issue by filippsen** [SuperblocksHQ/superblocks-lab#23]
## Environment/Browser
* node v10.5.0
* yarn 1.7.0
* npm 6.1.0

## Description
Deprecation warning during package installation (dependencies).

## Steps to reproduce
Simply run `npm install` or `yarn install`

## Expected result
Nothing.

## Actual result
```
(node:1) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
```

## Reproducible
Can be consistently reproduced.

## Suggestion for a fix
For studio code, in particular for `evm` and `solc` components, replace `new Buffer(...)` calls with `Buffer.from(...)`

**Comment by filippsen** [comment]
Unable to reproduce in current master (d28a401).

node v8.11.3
yarn 1.3.2

**Comment by javier-tarazaga** [comment]
Solved in #210
